src/bun.js/bindings/bindings.zig

```diff
@@ -2762,6 +2867,22 @@ pub const JSValue = enum(JSValueReprInt) {
         });
     }
 
+    pub fn asAnyPromise(
+        value: JSValue,
+    ) ?*AnyPromise {
+        if (value.asInternalPromise()) |promise| {
+            return &AnyPromise{
```

Comment:
This is a pointer to stack memory which will be an invalid pointer immediately after it is used

This will segfault

Reply:
changed from reference to value in https://github.com/oven-sh/bun/pull/1697/commits/0323676611ecdefe46d6e334d52dfc7360b2856e